<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageConfigBase</class>
 <widget class="QWidget" name="MessageConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="chkStatus">
     <property name="text">
      <string>Save &amp;status changes to history</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkOnline">
     <property name="text">
      <string>Open window if user go online</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpWindow">
     <property name="title">
      <string>New message</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="btnNoOpen">
        <property name="text">
         <string>&amp;No open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="btnMinimize">
        <property name="text">
         <string>Open and &amp;minimize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="btnRaise">
        <property name="text">
         <string>Open and &amp;raise</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="Spacer7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkStatus</tabstop>
  <tabstop>chkOnline</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
